docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,6253 @@
+{
```

Comment:
<p>Maybe rename this to:</p><p> </p><p>Computing the Dual Grid with Data.</p><p> </p><p>Can group together both the UxDataArray and UxDataset sections. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>